File: BabylonProject/Assets/Content/Mobile/MobileOccludeMaterial.ts
module PROJECT {
    /**
    * Babylon Script Component
    * @class MobileOccludeMaterial
    */
    export class MobileOccludeMaterial extends BABYLON.Toolkit.ScriptComponent {
        private applyToMaterial: boolean = true;

        protected awake(): void {
            if (this.applyToMaterial === true && this.transform instanceof BABYLON.AbstractMesh) {
                this.transform.material.disableColorWrite = true;
            }
        }
    }
}

File: BabylonProject/Assets/Content/Mobile/MobileOccludeMaterial.cs
using System;
using UnityEditor;
using UnityEngine;

/**
* Editor Script Component
* @class MobileOccluderMaterial
*/
[Babylon(Class="PROJECT.MobileOccludeMaterial"), AddComponentMenu(CanvasToolsStatics.CANVAS_TOOLS_MENU + "/Content Components/Mobile/Mobile Occlude Material", 2105)]
public class MobileOccludeMaterial : EditorScriptComponent
{
    [Tooltip("Apply occlusion to source material")]
    [Auto] public bool applyToMaterial = true;
}
